man/common/authorizations.md

```diff
@@ -1,4 +1,13 @@
-# Password Formatting
+# Authorization Formatting
+
+Authorization for use of an object in tpm2.0 can come in 3 different forms:
```

Comment:
We usually refer in the man page and docs as TPM 2.0 or TPM2.0, but never as tpm2.0.